docs/source/markdown/podman-port.1.md.in

```diff
@@ -15,13 +15,9 @@ List port mappings for the *container* or look up the public-facing port that is
 
 #### **--all**, **-a**
 
-List all known port mappings for running containers.  When using this option, you cannot pass any container names
-or private ports/protocols as filters.
+List all known port mappings for running containers.  When using this option container names or private ports/protocols as filters can not be used.
```

Comment:
Dropped the comma, please re-add: "when using this option COMMA container names ..."

Also, **cannot* is oneword.